apps/etterlatte-behandling/src/main/kotlin/BeanFactory.kt

```diff
@@ -24,14 +24,10 @@ import no.nav.etterlatte.behandling.foerstegangsbehandling.Foerstegangsbehandlin
 import no.nav.etterlatte.behandling.foerstegangsbehandling.FoerstegangsbehandlingService
 import no.nav.etterlatte.behandling.foerstegangsbehandling.RealFoerstegangsbehandlingService
 import no.nav.etterlatte.behandling.hendelse.HendelseDao
-import no.nav.etterlatte.behandling.klienter.BeregningKlient
-import no.nav.etterlatte.behandling.klienter.BeregningKlientImpl
 import no.nav.etterlatte.behandling.klienter.GrunnlagKlient
 import no.nav.etterlatte.behandling.klienter.GrunnlagKlientImpl
 import no.nav.etterlatte.behandling.klienter.VedtakKlient
 import no.nav.etterlatte.behandling.klienter.VedtakKlientImpl
```

Comment:
Skulle du fjerne vedtak og?

Reply:
Fra vedtak hentes saksbehandler, datofattet, datoattestert og attestant som vises i sidemenyen hele veien (de to siste bare etter attestering riktignok), så de forblir nok (hvert fall for nå). Så er spørsmålet om dette kanskje er data som bør ligge i behandling, men usikker på sistnevnte her